### PR TITLE
feat: design pass updates

### PR DESCRIPTION
**Pull request by shrugs**
fixes https://github.com/urbit/bridge/issues/208

### CODE REVIEW

File: src/views/Admin/AdminNetworkingKeys.js [hunk 1]
```diff
@@ -330,7 +330,8 @@ export default function AdminNetworkingKeys() {
         {completed && (
           <>
             <Grid.Item full as={NoticeBox} className="mb3">
-              You need this keyfile to authenticate with Arvo.
+              You need this keyfile to authenticate with Arvo. If you have
+              already booted Arvo, reboot with this new keyfile.
```
Comment (Fang-) [inline src/views/Admin/AdminNetworkingKeys.js]: This is only true in the `discontinuous` case.

To use new keys while remaining continuous, you use the `|rekey` command, pasting in the contents of your keyfile as a string. ie `|rekey '0w123.whatever`

That has its own slew of issues (it's kinda strict about when/how you can change keys), but certainly better than telling people to reboot their ships.

Proposed copy: "If you have already booted Arvo, run `|rekey '0wkeyfile.contents'`

I'd include a link to the docs, but that doesn't include this at all. cc @rmariani @baudtack

Comment (shrugs) [reply src/views/Admin/AdminNetworkingKeys.js]: I'm going to update to that copy but this needs more thought around the whole set of flows and documentation etc to be really handled.

File: src/lib/useInputs.js [hunk 2]
```diff
@@ -22,7 +22,12 @@ import { prependSig, convertToNumber } from 'lib/transformers';
 
 import InputSigil from 'components/InputSigil';
 
-const EXAMPLE_PRIVATE_KEY =
+const PLACEHOLDER_POINT = '~zod';
+const PLACEHOLDER_HD_PATH = DEFAULT_HD_PATH;
+const PLACEHOLDER_MNEMONIC = bip39.generateMnemonic();
+const PLACEHOLDER_TICKET = '~ragtyd-modwen-bostec-hinsep';
+const PLACEHOLDER_ADDRESS = '0x6DEfFb0caFDB11D175F123F6891AA64F01c24F7d';
+const PLACEHOLDER_PRIVATE_KEY =
```
Comment (Fang-) [inline src/lib/useInputs.js]: Proposed sample values:

```js
const PLACEHOLDER_POINT = '~sampel-ponnym';
const PLACEHOLDER_HD_PATH = DEFAULT_HD_PATH;
const PLACEHOLDER_MNEMONIC =
  'example crew supreme gesture quantum web media hazard theory mercy wing kitten';
const PLACEHOLDER_TICKET = '~sampel-ticlyt-migfun-falmel;
const PLACEHOLDER_ADDRESS = '0x12345abcdeDB11D175F123F6891AA64F01c24F7d';
const PLACEHOLDER_PRIVATE_KEY =
  '0x12345abcdee6beb2f323fab48b432925c9785808d33a6ca6d7ba00b45e9370c3';
```

**Comment by shrugs** [comment]
@Fang- not at my computer, can you make those changes and merge? lgtm

**Pull request merged**